cmake/Platform/Core/BoardFlags/CompilerFlagsSetter.cmake

```diff
@@ -1,4 +1,19 @@
 # ToDo: Comment
+
+function(_sanitize_quotes
+   cmd_line_var_
```

Comment:
2 questions:

1. Why is it defined on a separate line?
2. Why is there an underscore (_) at **the end** of the var's name? Is it your convention?

Reply:
In all my projects I use underscores to distinguish local variables from variables that are defined in parent scope. This only makes a difference in functions, though.

I will replace `cmd_line_var_` with `CMD_LINE_VARIABLE` to make it more conforming and verbose.